tests/test_weighted_pandas.py

```diff
@@ -182,8 +182,8 @@ def test_WeightedDataFrame_corrwith(frame):
     with pytest.raises(ValueError):
         frame.corrwith(unweighted[['A', 'B']])
 
-    with pytest.raises(ValueError):
-        unweighted.corrwith(frame.A)
+    # with pytest.raises(ValueError):
+    #     unweighted.corrwith(frame.A)
```

Comment:
Did this test originally pass?

Reply:
Yes, the *test* passes for `pandas<=1.4.4`, meaning that applying `corrwith` between a `WeightedDataFrame` and a `Series` will raise a `ValueError` because of incompatible indices.

Note, the other direction (a few lines prior to this one) still raises a `ValueError`.

Reply:
that seems like a pandas decision, and we should follow whatever it does with default dataframes.

Reply:
cleaned up accordingly in [b62f4ef](https://github.com/williamjameshandley/anesthetic/pull/238/commits/b62f4ef8cfa40a313e57b10695fd400d1a02f41b)